treeherder/log_parser/parsers.py

```diff
@@ -388,6 +388,7 @@ class ErrorParser(ParserBase):
         r"|^The web-page 'stop build' button was pressed"
         r"|.*\.js: line \d+, col \d+, Error -"
         r"|^\[taskcluster\] Error:"
+        r"|^\[[\w-]+:(?:error|exception)+\]"
```

Comment:
Yeah follow the setup steps at http://treeherder.readthedocs.org/installation.html (if all you want to do is run the tests, you can skip the `/etc/hosts` step) and then after a `vagrant ssh` run:
`py.test tests/model/test_error_summary.py`

For more details, see:
http://treeherder.readthedocs.org/common_tasks.html#running-the-tests